promql/testdata/aggregators.test

```diff
@@ -94,6 +196,28 @@ eval instant at 50m stdvar by (instance)(http_requests)
   {instance="0"} 50000
   {instance="1"} 50000
 
+# Over time functions
+eval instant at 50m avg_over_time(http_requests{group="production",job="api-server"}[1h])
+	{group="production", instance="0", job="api-server"} 50
+	{group="production", instance="1", job="api-server"} 100
+
+eval instant at 50m count_over_time(http_requests{group="production",job="api-server"}[1h])
```

Comment:
those _over_time belong in functions.test

Reply:
I actually just decided to remove them since all of them are represented in the functions.test file. There's nothing unique about them, so it's just additionally testing for no gain.